test/parallel/test-cli-eval.js

```diff
@@ -288,3 +288,11 @@ child.exec(
   common.mustSucceed((stdout) => {
     assert.strictEqual(stdout, '.mjs file\n');
   }));
+
+// Assert that `crypto` refers to `node:crypto` module in this context.
+child.exec(
+  `${nodejs} ` +
+  '-e "console.log(crypto.randomBytes(16).toString(\'hex\'))"',
+  common.mustSucceed((stdout) => {
+    assert.match(stdout, /[0-9a-f]{32}/i);
+  }));
```

Comment:
```suggestion
// Assert that `crypto` refers to `node:crypto` module in this context.
child.exec(
  `${nodejs} ` +
  '-p "crypto === require(\'crypto\')"',
  common.mustSucceed((stdout) => {
    assert.match(stdout, /true/);
  }));
```

Reply:
I wanted to test an actual use case, even if `crypto === require('crypto')` is false, as long as the current test case works, we're ok I'd say.